requirements/edx/base.txt

```diff
@@ -103,16 +103,16 @@ edx-django-oauth2-provider==1.3.5
 edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
 edx-django-utils==1.0.3
-edx-drf-extensions==2.2.1
-edx-enterprise==1.6.0
+edx-drf-extensions==2.3.0
+edx-enterprise==1.6.2
 edx-i18n-tools==0.4.8
 edx-milestones==0.1.13
 edx-oauth2-provider==1.2.2
 edx-opaque-keys[django]==0.4.4
-edx-organizations==2.0.2
+edx-organizations==2.0.3
```

Comment:
reviewers: this adds history to the Organization model.

Reply:
I think you will need to install this locally and re-run `makemigrations` to get the migration that actually adds the new table?

Reply:
That migration is inside the edx-organizations repo.

Reply:
Hm, not sure what weird case I was thinking of where we needed the migration in both, but 👍